webapp-django/crashstats/crashstats/forms.py

```diff
@@ -235,6 +235,16 @@ def clean_v(self):
             )
         return versions
 
+    def clean_date_end(self):
+        value = self.cleaned_data['date_end']
+        if value:
+            now = datetime.datetime.utcnow()
+            if not isinstance(value, datetime.datetime):
```

Comment:
Why not just ``if isinstance(value, datetime.date):``? It's simpler to understand and I think strictly equivalent here.